platform/src/main/java/org/hillview/table/ObjectArrayColumn.java

```diff
@@ -178,4 +179,9 @@ public static ObjectArrayColumn mergeColumns(final IColumn left, final IColumn r
         }
         return merged;
     }
+
+    @Override
+    public long hashCode64(int rowIndex, long seed) {
+        return LongHashFunction.xx(seed).hashLong((long) this.asDouble(rowIndex,null));
```

Comment:
It looks wrong to convert the double to long, this loses information.
I think you want to use Double.doubleToRawLongBits.